### PR TITLE
ci-operator: Scale an extra replica instance for metering jobs.

### PR DESCRIPTION
**Pull request by timflannagan**

### CODE REVIEW

File: ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-feature.yaml [hunk 1]
```diff
@@ -60,11 +60,7 @@ tests:
     set -x
     # Increase the size of the machines so that there are machines large enough
     # to run Presto, and the full metering stack.
-    kubectl -n openshift-machine-api get machinesets -o name | grep worker | cut -d/ -f2 | xargs -I{} kubectl -n openshift-machine-api patch machineset {} --type json -p '[{"op": "replace", "path": "/spec/template/spec/providerSpec/value/instanceType", "value": "m4.xlarge"}]'
-    # delete the workers so that larger machines replace it
-    # wait=false because the machines never actually get deleted from the
-    # API
-    kubectl -n openshift-machine-api get machines -o name | grep worker | xargs kubectl -n openshift-machine-api delete --wait=false
+    kubectl -n openshift-machine-api get machinesets -o name | grep worker | cut -d/ -f2 | xargs -I{} kubectl -n openshift-machine-api scale machineset {} --replicas 3
```
Comment (bentito) [inline ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-feature.yaml:63]: Where did the actual ref to the type/size go, like `m4.xlarge`?

Comment (timflannagan1) [reply ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-feature.yaml:63]: CI bumped the default instance size to `m4.xlarge` so this modification isn't necessary anymore.

Comment (bentito) [reply ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-feature.yaml:63]: cool

**Comment by timflannagan1** [comment]
/override ci/rehearse/operator-framework/operator-metering/release-4.3

**Pull request merged**